Jenkinsfile

```diff
@@ -3,7 +3,7 @@
 library("govuk")
 
 def rubyVersions = ["2.6", "2.7"]
```

Comment:
We should probably add 3.0 in here too to test Ruby 3 (I don't think we have 3.1 available yet)

Reply:
I did wonder about doing so too, but it seemed less pressing. Happy to add that too though.